overlays/openstack.yaml

```
@@ -0,0 +1,17 @@
+applications:
+  openstack-integrator:
+    charm: openstack-integrator
+    num_units: 1
+    trust: true
+    base: ubuntu@22.04
+  openstack-cloud-controller:
+    charm: openstack-cloud-controller
+    base: ubuntu@22.04
+  cinder-csi:
+    charm: cinder-csi
+    base: ubuntu@22.04
+relations:
+  - [openstack-cloud-controller:kube-control,            k8s:kube-control]
```

cinder-csi needs `kube-control` too

```suggestion
  - [openstack-cloud-controller:kube-control,            k8s:kube-control]
  - [cinder-csi:kube-control,                            k8s:kube-control]
```